homeassistant/components/onewire/sensor.py

```diff
@@ -195,12 +224,14 @@
 )
 
 
-def hb_info_from_type(dev_type="std"):
+def get_sensor_types(device_sub_type="std"):
     """Return the proper info array for the device type."""
-    if "std" in dev_type:
+    if "std" in device_sub_type:
         return DEVICE_SENSORS
```

Comment:
Remove these two lines:

```suggestion
    if "std" in device_sub_type:
        return DEVICE_SENSORS
```

Reply:
Good comments. Much cleaner now.